bitnami/Makefile

```diff
@@ -6,8 +6,9 @@ PG_VER=pg15
 CITUS_VERSION="11.2.0"
 POSTGIS_MAJOR=3
 PG_MAJOR=15
-POSTGIS_VERSION=3.3.2+dfsg-1.pgdg110+1
+POSTGIS_VERSION=3.3.3+dfsg-1~exp1.pgdg110+1
```

Comment:
What does `pgdg110` refer to ?

Reply:
`pgdg110` is [ PostgreSQL Global Development Group apt repository](https://www.postgresql.org/about/news/pgdg-apt-repository-for-debianubuntu-1432/) for Debian 11

Reply:
`bitnami/postgresql` images uses `minideb`. Minideb is a minimalist Debian-based image built specifically to be used as a base image for containers(https://github.com/bitnami/minideb). Please confirm compatibility.

Reply:
The smoke test is passing https://github.com/Samagra-Development/WarpSQL/actions/runs/5142018741/jobs/9255221212?pr=40#step:5:103
Can you provide more details or specify any additional tests you would like me to perform?